test/system/500-networking.bats

```diff
@@ -317,14 +317,16 @@ load helpers.network
     run curl -s -S $SERVER/index.txt
     is "$output" "$random_1" "curl 127.0.0.1:/index.txt"
 
-    # rootless cannot modify iptables
+    # rootless cannot modify the host firewall
     if ! is_rootless; then
-        # flush the port forwarding iptable rule here
-        chain="CNI-HOSTPORT-DNAT"
-        if is_netavark; then
-            chain="NETAVARK-HOSTPORT-DNAT"
-        fi
-        run iptables -t nat -F "$chain"
+        # for debugging only
+        iptables -t nat -nvL || true
+        nft list ruleset     || true
+
+        # flush the firewall rule here to break port forwarding
+        # netavark can use either iptables or nftables, so try flushing both
+        iptables -t nat -F "NETAVARK-HOSTPORT-DNAT" || true
+        nft delete table inet netavark              || true
```

Comment:
If nft is the desired default, shouldn't that be the first one run?

And, what happens on systems where both are installed? Would it be safer to do this instead?
```bash
    nft delete table inet netavark \
        || iptables -t nat -F "..." \
        || true
```

Reply:
That is what I tried before as well, there is big bug in system there that use CONTAINERS_CONF which will mean they ignore the firewall driver opt.

I had the order iptables || nft which does not work, using nft || iptables would work but not if somebody inverts the order one day where the containers.conf forces iptables and the normal the default is nft. Thus I decied to just always flush both.

In general using CONTAINERS_CONF in system tests seems wrong but well there is a test that excpliclty checks how CONTAINERS_CONF works so it is not like I can just remove/fix it either...